app/decorators/user_decorator.rb

```diff
@@ -23,11 +23,11 @@ def confirmation_period
 
   def first_sentence_for_confirmation_email
     if user.reset_requested_at
-      'Your Upaya account has been reset by a tech support representative. ' \
+      "Your #{APP_NAME} account has been reset by a tech support representative. " \
       'In order to continue, you must confirm your email address.'
     else
       "To finish #{user.confirmed_at ? 'updating' : 'creating'} your " \
-      'Upaya Account, you must confirm your email address.'
+      "#{APP_NAME} Account, you must confirm your email address."
```

Comment:
While we're making these changes, should we also make the spelling of `account` consistent? Should it be lowercase all the time?